app.rb

```diff
@@ -91,18 +123,92 @@ def list_people
     end
   end
 
-  def list_rentals
-    if @rentals.empty?
-      puts 'There are no rentals to show'
-    else
-      puts 'ID of person: '
-      person_id = gets.chomp.to_i
-      puts 'Rentals: '
-      @rentals.each do |rental|
-        if person_id == rental.person.id
-          puts "Date: #{rental.date}, Book '#{rental.book.title}' by #{rental.book.author}"
-        end
+  # def list_rentals
+  #   if @rentals.empty?
+  #     puts 'There are no rentals to show'
+  #   else
+  #     puts 'ID of person: '
+  #     person_id = gets.chomp.to_i
+  #     puts 'Rentals: '
+  #     @rentals.each do |rental|
+  #       if person_id == rental.person.id
+  #         puts "Date: #{rental.date}, Book '#{rental.book.title}' by #{rental.book.author}"
+  #       end
```

Comment:
- [Required] I think it would be nice if you pieces of commented code from your code base, commented code makes your project look messy. If you need to review a previous version of your code, you can always use git history. Please remove this.